webapp/channels/src/components/admin_console/schema_admin_settings.tsx

```diff
@@ -203,46 +203,6 @@ export class SchemaAdminSettings extends React.PureComponent<Props, State> {
         }
     };
 
-    getConfigFromState(config: Partial<AdminConfig>) {
```

Comment:
This and the other functions are now exported below so we can use them in the ldap wizard too.